site/en/fugu-showcase/index.njk

```diff
@@ -63,6 +63,16 @@ alt: Blowfish swarm swimming in the ocean.
     </div>
   </div>
 </div>
+
+<div class="display-none" id="i_just_found_the_app">
+  {{ 'i18n.fugu_showcase.i_just_found_the_app' | i18n(locale) }}
+</div>
+<div class="display-none" id="it_uses_these_fugu_apis">
+  {{ 'i18n.fugu_showcase.it_uses_these_fugu_apis' | i18n(locale) }}
+</div>
+<div class="display-none" id="via_the_fugu_showcase">
+  {{ 'i18n.fugu_showcase.via_the_fugu_showcase' | i18n(locale) }}
+</div>
```

Comment:
I'd vote to use a `<script id="fugu-showcase-i18n" type="application/json"></script>` or something instead.

Reply:
Will do.